src/Data/Rational/Properties.agda

```diff
@@ -810,6 +814,23 @@ private
 *-distrib-+ : _*_ DistributesOver _+_
 *-distrib-+ = *-distribˡ-+ , *-distribʳ-+
 
+*-inverseˡ : ∀ p {p≢0 : ∣ ↥ p ∣ ≢0} → 1/_ p {p≢0} * p ≡ 1ℚ
+*-inverseˡ p {p≢0} = toℚᵘ-injective (begin-equality
+  toℚᵘ (1/ p * p)
+    ≈⟨ toℚᵘ-homo-* (1/ p) p ⟩
+  toℚᵘ (1/ p) ℚᵘ.* toℚᵘ p
+    ≈⟨ ℚᵘ.*-cong (toℚᵘ-homo-1/ p {p≢0}) ℚᵘ.≃-refl ⟩
```

Comment:
This could be simplified by adding `*-congˡ` and `*-congʳ` to `Unnormalised.Properties`. Which would probably then allow you to move the justifications onto the same line as the terms without breaking the line limit?